libs/ballot-interpreter-vx/src/cli/index.ts

```diff
@@ -140,40 +45,34 @@ export async function main(
   stdout: NodeJS.WritableStream,
   stderr: NodeJS.WritableStream
 ): Promise<number> {
-  try {
-    const commandOptions = await parseOptions(args);
-
-    switch (commandOptions.command) {
-      case 'help':
-        return await helpCommand.run(commandOptions.options, stdin, stdout);
+  const commands = getCommands();
+  const globalOptionsResult = parseGlobalOptions(args);
 
-      case 'interpret':
-        return await interpretCommand.run(
-          commandOptions.options,
-          stdin,
-          stdout
-        );
+  if (globalOptionsResult.isErr()) {
+    stderr.write(`error: ${globalOptionsResult.err().message}\n`);
+    return -1;
```

Comment:
never seen a negative exit code before. is this a thing?

Reply:
It becomes 255. I've been using it to mean "I couldn't understand what you told me to do", i.e. a 400 in HTTP-speak.

```sh
❯ sh -c 'exit -1'
❯ echo $?
255
```